README.md

```diff
@@ -210,6 +211,27 @@ For dex.decred.org, reach out
 [on Element](https://matrix.to/#/!mlRZqBtfWHrcmgdTWB:decred.org?via=decred.org&via=matrix.org&via=planetdecred.org)
 to appeal.
 
+## Fees
+
+The dex does not charge trading fees, but users have to pay on-chain transaction
+fees. Unlike on centralized exchanges where users can trade any arbitrary amount
+of funds, orders on the dex must be made in increments of a specified lot size.
+For example, if the lot size for the DCR/BTC market is 40 DCR, users can
+make orders to buy or sell DCR in increments of 40.
+
+You might be lucky and have your entire order matched by one other user. In
+this case, you will have to pay 2 transaction fees: one for sending funds on
+one chain, and one for receiving funds on the other. In the worst case, each
+lot of your order is matched by a different user, and you will pay a separate
+sending and recieving transaction for each match. Check the 
+[dex specification](https://github.com/decred/dcrdex/blob/master/spec/atomic.mediawiki)
+for more details about how atomic swaps work.
+
+The client contains an option to pre-split UTXOs in order to not lock up
+more funds than necessary for the duration of an atomic swap. This option will
```

Comment:
~~The client contains an option to pre-split UTXOs in order to not lock up more funds than necessary for the duration of an atomic swap.~~

Wallets for UTXO-based assets provide an option to use an additional transaction per order to pre-size outputs and prevent over-locking of funds that might otherwise occur until the first swap is broadcast for the order.

---

But really, I think this is more detail than is appropriate for the README. We have a wiki for this kind of stuff.